docs/quickstarts/php-quickstart.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: PHP Quickstart
+---
+
+<h1 class="page-header">PHP Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in PHP with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+* `php`: version 5.5 or higher, 7.0 or higher
+* `pecl`: version 1.9 or higher
+* `composer`
+
+Note that currently you can only create clients in PHP for gRPC services -
+you can find out how to create gRPC servers in our other tutorials,
+e.g. [Node.js](node-quickstart.md).
+
+### Install gRPC
+
+Install gRPC:
+
+```sh
+  $ [sudo] pecl install grpc
+```
+
+### Install Protobuf-PHP
+
+You will need to install the protocol buffer compiler `protoc`
+and the special plugin for generating server and client code
+from `.proto` service definitions. For the first part of our
+quickstart example, we've already generated the server and client
+stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{site.data.config.grpc_release_branch}}/examples/protos/helloworld.proto),
+but you'll need the tools for the rest of our quickstart, as well as later
+tutorials and your own projects.
+
+To install Protobuf-PHP, run:
+
+```sh
+  $ git clone https://github.com/stanley-cheung/Protobuf-PHP
+  $ cd Protobuf-PHP
+  $ rake pear:package version=1.0
+  $ [sudo] pear install Protobuf-1.0.tgz
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart and other tutorials):
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" PHP example:
+  $ cd grpc/examples/php
+  $ composer install
+```
+
+## Run a gRPC application
+
+From the `examples/node/static_codegen` directory:
+
+1. Run the server
+
+   ```sh
+   $ node greeter_server.js
+   ```
+
+From the `examples/php` directory:
+
+2. In another terminal, run the client
```

Comment:
I'd maybe rejig this a bit, so

In another terminal, from the `examples/php` directory:

2. Run the client: